pkg/agent/datamodel/helper.go

```diff
@@ -22,7 +22,9 @@ func ValidateDNSPrefix(dnsName string) error {
 		return err
 	}
 	if !re.MatchString(dnsName) {
-		return errors.Errorf("DNSPrefix '%s' is invalid. The DNSPrefix must contain between 3 and 45 characters and can contain only letters, numbers, and hyphens.  It must start with a letter and must end with a letter or a number. (length was %d)", dnsName, len(dnsName))
+		return errors.Errorf("DNSPrefix '%s' is invalid. The DNSPrefix must contain between 3 and 45 characters"+
```

Comment:
nit: extract the error template to a multiline string outside the function body?

Reply:
golint will probably complain of a global variable if I take it outside of the function body. But I could define it as a variable inside the function?

Reply:
Also can I make a multiline string in go using ` when there are variables in the string?